### ISSUE TITLE
Auth HTTP Plugin: ACL request parameter `%A` is always 2

### ISSUE THREAD

**Issue by PMExtra** [emqx/emqx#2400]
#### Environment

Docker Image: emqx/emqx:v3.1-beta.3

#### Description

Auth HTTP Plugin: ACL request parameter `%A` is always 2.

According to the [document](https://developer.emqx.io/docs/emq/v3/cn/plugins.html#emqx-auth-http-http):
```
## Variables:
##  - %A: 1 | 2, 1 = sub, 2 = pub
##  - %u: username
##  - %c: clientid
##  - %a: ipaddress
##  - %t: topic
auth.http.acl_req = http://127.0.0.1:8080/mqtt/acl
auth.http.acl_req.method = get
auth.http.acl_req.params = access=%A,username=%u,clientid=%c,ipaddr=%a,topic=%t
```

But in fact, the `access` parameter is always `2` whether the client sub or pub.

I have tested and stably reproduced this bug in EMQ Dashboard Websockets.

**Comment by phanimahesh** [comment]
Noticed and fixed in https://github.com/emqx/emqx/pull/2378

Expect the fix to land in next release. Apologies for the bug.

**Comment by HJianBo** [comment]
Yes, We'll release this fix today! You can check it at v3.1.rc.1 later.

Thanks for your feedback.

**Comment by PMExtra** [comment]
Fine. Thanks you all.
